vita3k/config/src/config.cpp

```diff
@@ -133,7 +133,7 @@ static ExitCode parse(Config &cfg, const fs::path &load_path, const std::string
     get_yaml_value(config_node, "color-surface-debug", &cfg.color_surface_debug, false);
     get_yaml_value(config_node, "hardware-flip", &cfg.hardware_flip, false);
     get_yaml_value(config_node, "performance-overlay", &cfg.performance_overlay, false);
-    get_yaml_value(config_node, "backend-renderer", &cfg.backend_renderer, std::string("OpenGL"));
+    get_yaml_value(config_node, "backend-renderer", &cfg.backend_renderer, static_cast<std::string>("OpenGL"));
```

Comment:
Effectively, this casting is the same as `std::string()`. Use `std::string()` to make it consistent with the rest of the file.

Reply:
i have set like this for get like other have static<int> ^^

Reply:
No need for strings, we have a constructor!

Reply:
forget finish line lol for get same other with static<int> so here string haha

Reply:
yes, i have just understund why other have static cast for int, stupid me lol